docs/book/src/topics/vpc/load-balancer.md

```diff
@@ -0,0 +1,23 @@
+## Provision workload Cluster with Load Balancer in IBM Cloud VPC
+### This feature is currently in experimental stage
+
+### Steps
+
+- To deploy a VPC workload cluster with Load Balancer, create a cluster configuration with the [load balancer template](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/main/templates/cluster-template-load-balancer.yaml)
```

Comment:
Pointing to the code in the webpages is not a great idea, this can break anytime, may be we can remove the gh ref here.

Reply:
Removed ref.